### PR TITLE
update to use azure release workflow

### PR DESCRIPTION
**Pull request by davidgamero**
update this repo to use the Azure release workflow for changelog-driven releases

### CODE REVIEW

File: .github/workflows/release-pr.yml [hunk 1]
```diff
@@ -9,6 +9,6 @@ on:
 
 jobs:
    release-pr:
-      uses: OliverMKing/javascript-release-workflow/.github/workflows/release-pr.yml@main
+      uses: Azure/action-release-workflows/.github/workflows/release_js_project.yaml@a705b2ab6a3ee889f2b0d925ad0bd2f9eb733ce6
```
Comment (jaiveerk) [inline .github/workflows/release-pr.yml:12]: is there no tag for this hash? feel like a human-readable version would be cleaner/easier to maintain in the future

**Review by jaiveerk — DISMISSED** [review]

File: .github/workflows/release-pr.yml (hunk 1 above)
Comment (davidgamero) [reply .github/workflows/release-pr.yml:12]: yeah we can add tags in the future, but right now we've been updating the hash each time so the tags would just be aliases per hash

Comment (OliverMKing) [reply .github/workflows/release-pr.yml:12]: hash is the best practice. tags are not as secure. We should continue to use sha for all action pinning

File: .github/workflows/release-pr.yml [hunk 2]
```diff
@@ -9,6 +9,6 @@ on:
...
       with:
          release: ${{ github.event.inputs.release }}
```
Comment (OliverMKing) [inline .github/workflows/release-pr.yml:14]: you need to pass in the changelogPath as input not the release

**Review by OliverMKing — CHANGES_REQUESTED** [review]

File: .github/workflows/release-pr.yml (hunk 2 above)
Comment (davidgamero) [reply .github/workflows/release-pr.yml]: ty updated

**Review by OliverMKing — DISMISSED** [review]

**Pull request closed without merging**